[25-Feb-2022 08:11:05 Europe/Paris] Drupal\Core\Database\DatabaseAccessDeniedException: SQLSTATE[HY000] [1045] Access denied for user 'wghm7504_root'@'localhost' (using password: YES) in /home/wghm7504/falzon9/core/lib/Drupal/Core/Database/Driver/mysql/Connection.php on line 197 #0 /home/wghm7504/falzon9/core/lib/Drupal/Core/Database/Database.php(387): Drupal\Core\Database\Driver\mysql\Connection::open(Array)
#1 /home/wghm7504/falzon9/core/lib/Drupal/Core/Database/Database.php(169): Drupal\Core\Database\Database::openConnection('default', 'default')
#2 [internal function]: Drupal\Core\Database\Database::getConnection('default')
#3 /home/wghm7504/falzon9/core/lib/Drupal/Component/DependencyInjection/PhpArrayContainer.php(79): call_user_func_array('Drupal\\Core\\Dat...', Array)
#4 /home/wghm7504/falzon9/core/lib/Drupal/Component/DependencyInjection/Container.php(176): Drupal\Component\DependencyInjection\PhpArrayContainer->createService(Array, 'database')
#5 /home/wghm7504/falzon9/core/lib/Drupal/Component/DependencyInjection/PhpArrayContainer.php(212): Drupal\Component\DependencyInjection\Container->get('database', 1)
#6 /home/wghm7504/falzon9/core/lib/Drupal/Component/DependencyInjection/PhpArrayContainer.php(62): Drupal\Component\DependencyInjection\PhpArrayContainer->resolveServicesAndParameters(Array)
#7 /home/wghm7504/falzon9/core/lib/Drupal/Component/DependencyInjection/Container.php(176): Drupal\Component\DependencyInjection\PhpArrayContainer->createService(Array, 'cache.container')
#8 /home/wghm7504/falzon9/core/lib/Drupal/Core/DrupalKernel.php(549): Drupal\Component\DependencyInjection\Container->get('cache.container')
#9 /home/wghm7504/falzon9/core/lib/Drupal/Core/DrupalKernel.php(894): Drupal\Core\DrupalKernel->getCachedContainerDefinition()
#10 /home/wghm7504/falzon9/core/lib/Drupal/Core/DrupalKernel.php(472): Drupal\Core\DrupalKernel->initializeContainer()
#11 /home/wghm7504/falzon9/core/lib/Drupal/Core/DrupalKernel.php(707): Drupal\Core\DrupalKernel->boot()
#12 /home/wghm7504/falzon9/index.php(19): Drupal\Core\DrupalKernel->handle(Object(Symfony\Component\HttpFoundation\Request))
#13 {main}
[25-Feb-2022 08:11:05 Europe/Paris] Drupal\Core\Database\DatabaseAccessDeniedException: SQLSTATE[HY000] [1045] Access denied for user 'wghm7504_root'@'localhost' (using password: YES) in /home/wghm7504/falzon9/core/lib/Drupal/Core/Database/Driver/mysql/Connection.php on line 197 #0 /home/wghm7504/falzon9/core/lib/Drupal/Core/Database/Database.php(387): Drupal\Core\Database\Driver\mysql\Connection::open(Array)
#1 /home/wghm7504/falzon9/core/lib/Drupal/Core/Database/Database.php(169): Drupal\Core\Database\Database::openConnection('default', 'default')
#2 [internal function]: Drupal\Core\Database\Database::getConnection('default')
#3 /home/wghm7504/falzon9/core/lib/Drupal/Component/DependencyInjection/PhpArrayContainer.php(79): call_user_func_array('Drupal\\Core\\Dat...', Array)
#4 /home/wghm7504/falzon9/core/lib/Drupal/Component/DependencyInjection/Container.php(176): Drupal\Component\DependencyInjection\PhpArrayContainer->createService(Array, 'database')
#5 /home/wghm7504/falzon9/core/lib/Drupal/Component/DependencyInjection/PhpArrayContainer.php(212): Drupal\Component\DependencyInjection\Container->get('database', 1)
#6 /home/wghm7504/falzon9/core/lib/Drupal/Component/DependencyInjection/PhpArrayContainer.php(62): Drupal\Component\DependencyInjection\PhpArrayContainer->resolveServicesAndParameters(Array)
#7 /home/wghm7504/falzon9/core/lib/Drupal/Component/DependencyInjection/Container.php(176): Drupal\Component\DependencyInjection\PhpArrayContainer->createService(Array, 'cache.container')
#8 /home/wghm7504/falzon9/core/lib/Drupal/Core/DrupalKernel.php(549): Drupal\Component\DependencyInjection\Container->get('cache.container')
#9 /home/wghm7504/falzon9/core/lib/Drupal/Core/DrupalKernel.php(894): Drupal\Core\DrupalKernel->getCachedContainerDefinition()
#10 /home/wghm7504/falzon9/core/lib/Drupal/Core/DrupalKernel.php(472): Drupal\Core\DrupalKernel->initializeContainer()
#11 /home/wghm7504/falzon9/core/lib/Drupal/Core/DrupalKernel.php(707): Drupal\Core\DrupalKernel->boot()
#12 /home/wghm7504/falzon9/index.php(19): Drupal\Core\DrupalKernel->handle(Object(Symfony\Component\HttpFoundation\Request))
#13 {main}
[25-Feb-2022 09:32:11 Europe/Paris] Drupal\Core\Database\DatabaseAccessDeniedException: SQLSTATE[HY000] [1045] Access denied for user 'wghm7504_root'@'localhost' (using password: YES) in /home/wghm7504/falzon9/core/lib/Drupal/Core/Database/Driver/mysql/Connection.php on line 197 #0 /home/wghm7504/falzon9/core/lib/Drupal/Core/Database/Database.php(387): Drupal\Core\Database\Driver\mysql\Connection::open(Array)
#1 /home/wghm7504/falzon9/core/lib/Drupal/Core/Database/Database.php(169): Drupal\Core\Database\Database::openConnection('default', 'default')
#2 [internal function]: Drupal\Core\Database\Database::getConnection('default')
#3 /home/wghm7504/falzon9/core/lib/Drupal/Component/DependencyInjection/PhpArrayContainer.php(79): call_user_func_array('Drupal\\Core\\Dat...', Array)
#4 /home/wghm7504/falzon9/core/lib/Drupal/Component/DependencyInjection/Container.php(176): Drupal\Component\DependencyInjection\PhpArrayContainer->createService(Array, 'database')
#5 /home/wghm7504/falzon9/core/lib/Drupal/Component/DependencyInjection/PhpArrayContainer.php(212): Drupal\Component\DependencyInjection\Container->get('database', 1)
#6 /home/wghm7504/falzon9/core/lib/Drupal/Component/DependencyInjection/PhpArrayContainer.php(62): Drupal\Component\DependencyInjection\PhpArrayContainer->resolveServicesAndParameters(Array)
#7 /home/wghm7504/falzon9/core/lib/Drupal/Component/DependencyInjection/Container.php(176): Drupal\Component\DependencyInjection\PhpArrayContainer->createService(Array, 'cache.container')
#8 /home/wghm7504/falzon9/core/lib/Drupal/Core/DrupalKernel.php(549): Drupal\Component\DependencyInjection\Container->get('cache.container')
#9 /home/wghm7504/falzon9/core/lib/Drupal/Core/DrupalKernel.php(894): Drupal\Core\DrupalKernel->getCachedContainerDefinition()
#10 /home/wghm7504/falzon9/core/lib/Drupal/Core/DrupalKernel.php(472): Drupal\Core\DrupalKernel->initializeContainer()
#11 /home/wghm7504/falzon9/core/lib/Drupal/Core/DrupalKernel.php(707): Drupal\Core\DrupalKernel->boot()
#12 /home/wghm7504/falzon9/index.php(19): Drupal\Core\DrupalKernel->handle(Object(Symfony\Component\HttpFoundation\Request))
#13 {main}
[25-Feb-2022 09:32:12 Europe/Paris] Drupal\Core\Database\DatabaseAccessDeniedException: SQLSTATE[HY000] [1045] Access denied for user 'wghm7504_root'@'localhost' (using password: YES) in /home/wghm7504/falzon9/core/lib/Drupal/Core/Database/Driver/mysql/Connection.php on line 197 #0 /home/wghm7504/falzon9/core/lib/Drupal/Core/Database/Database.php(387): Drupal\Core\Database\Driver\mysql\Connection::open(Array)
#1 /home/wghm7504/falzon9/core/lib/Drupal/Core/Database/Database.php(169): Drupal\Core\Database\Database::openConnection('default', 'default')
#2 [internal function]: Drupal\Core\Database\Database::getConnection('default')
#3 /home/wghm7504/falzon9/core/lib/Drupal/Component/DependencyInjection/PhpArrayContainer.php(79): call_user_func_array('Drupal\\Core\\Dat...', Array)
#4 /home/wghm7504/falzon9/core/lib/Drupal/Component/DependencyInjection/Container.php(176): Drupal\Component\DependencyInjection\PhpArrayContainer->createService(Array, 'database')
#5 /home/wghm7504/falzon9/core/lib/Drupal/Component/DependencyInjection/PhpArrayContainer.php(212): Drupal\Component\DependencyInjection\Container->get('database', 1)
#6 /home/wghm7504/falzon9/core/lib/Drupal/Component/DependencyInjection/PhpArrayContainer.php(62): Drupal\Component\DependencyInjection\PhpArrayContainer->resolveServicesAndParameters(Array)
#7 /home/wghm7504/falzon9/core/lib/Drupal/Component/DependencyInjection/Container.php(176): Drupal\Component\DependencyInjection\PhpArrayContainer->createService(Array, 'cache.container')
#8 /home/wghm7504/falzon9/core/lib/Drupal/Core/DrupalKernel.php(549): Drupal\Component\DependencyInjection\Container->get('cache.container')
#9 /home/wghm7504/falzon9/core/lib/Drupal/Core/DrupalKernel.php(894): Drupal\Core\DrupalKernel->getCachedContainerDefinition()
#10 /home/wghm7504/falzon9/core/lib/Drupal/Core/DrupalKernel.php(472): Drupal\Core\DrupalKernel->initializeContainer()
#11 /home/wghm7504/falzon9/core/lib/Drupal/Core/DrupalKernel.php(707): Drupal\Core\DrupalKernel->boot()
#12 /home/wghm7504/falzon9/index.php(19): Drupal\Core\DrupalKernel->handle(Object(Symfony\Component\HttpFoundation\Request))
#13 {main}
[25-Feb-2022 09:33:15 Europe/Paris] Drupal\Core\Database\DatabaseAccessDeniedException: SQLSTATE[HY000] [1045] Access denied for user 'wghm7504_root'@'localhost' (using password: YES) in /home/wghm7504/falzon9/core/lib/Drupal/Core/Database/Driver/mysql/Connection.php on line 197 #0 /home/wghm7504/falzon9/core/lib/Drupal/Core/Database/Database.php(387): Drupal\Core\Database\Driver\mysql\Connection::open(Array)
#1 /home/wghm7504/falzon9/core/lib/Drupal/Core/Database/Database.php(169): Drupal\Core\Database\Database::openConnection('default', 'default')
#2 [internal function]: Drupal\Core\Database\Database::getConnection('default')
#3 /home/wghm7504/falzon9/core/lib/Drupal/Component/DependencyInjection/PhpArrayContainer.php(79): call_user_func_array('Drupal\\Core\\Dat...', Array)
#4 /home/wghm7504/falzon9/core/lib/Drupal/Component/DependencyInjection/Container.php(176): Drupal\Component\DependencyInjection\PhpArrayContainer->createService(Array, 'database')
#5 /home/wghm7504/falzon9/core/lib/Drupal/Component/DependencyInjection/PhpArrayContainer.php(212): Drupal\Component\DependencyInjection\Container->get('database', 1)
#6 /home/wghm7504/falzon9/core/lib/Drupal/Component/DependencyInjection/PhpArrayContainer.php(62): Drupal\Component\DependencyInjection\PhpArrayContainer->resolveServicesAndParameters(Array)
#7 /home/wghm7504/falzon9/core/lib/Drupal/Component/DependencyInjection/Container.php(176): Drupal\Component\DependencyInjection\PhpArrayContainer->createService(Array, 'cache.container')
#8 /home/wghm7504/falzon9/core/lib/Drupal/Core/DrupalKernel.php(549): Drupal\Component\DependencyInjection\Container->get('cache.container')
#9 /home/wghm7504/falzon9/core/lib/Drupal/Core/DrupalKernel.php(894): Drupal\Core\DrupalKernel->getCachedContainerDefinition()
#10 /home/wghm7504/falzon9/core/lib/Drupal/Core/DrupalKernel.php(472): Drupal\Core\DrupalKernel->initializeContainer()
#11 /home/wghm7504/falzon9/core/lib/Drupal/Core/DrupalKernel.php(707): Drupal\Core\DrupalKernel->boot()
#12 /home/wghm7504/falzon9/index.php(19): Drupal\Core\DrupalKernel->handle(Object(Symfony\Component\HttpFoundation\Request))
#13 {main}
[25-Feb-2022 09:33:15 Europe/Paris] Drupal\Core\Database\DatabaseAccessDeniedException: SQLSTATE[HY000] [1045] Access denied for user 'wghm7504_root'@'localhost' (using password: YES) in /home/wghm7504/falzon9/core/lib/Drupal/Core/Database/Driver/mysql/Connection.php on line 197 #0 /home/wghm7504/falzon9/core/lib/Drupal/Core/Database/Database.php(387): Drupal\Core\Database\Driver\mysql\Connection::open(Array)
#1 /home/wghm7504/falzon9/core/lib/Drupal/Core/Database/Database.php(169): Drupal\Core\Database\Database::openConnection('default', 'default')
#2 [internal function]: Drupal\Core\Database\Database::getConnection('default')
#3 /home/wghm7504/falzon9/core/lib/Drupal/Component/DependencyInjection/PhpArrayContainer.php(79): call_user_func_array('Drupal\\Core\\Dat...', Array)
#4 /home/wghm7504/falzon9/core/lib/Drupal/Component/DependencyInjection/Container.php(176): Drupal\Component\DependencyInjection\PhpArrayContainer->createService(Array, 'database')
#5 /home/wghm7504/falzon9/core/lib/Drupal/Component/DependencyInjection/PhpArrayContainer.php(212): Drupal\Component\DependencyInjection\Container->get('database', 1)
#6 /home/wghm7504/falzon9/core/lib/Drupal/Component/DependencyInjection/PhpArrayContainer.php(62): Drupal\Component\DependencyInjection\PhpArrayContainer->resolveServicesAndParameters(Array)
#7 /home/wghm7504/falzon9/core/lib/Drupal/Component/DependencyInjection/Container.php(176): Drupal\Component\DependencyInjection\PhpArrayContainer->createService(Array, 'cache.container')
#8 /home/wghm7504/falzon9/core/lib/Drupal/Core/DrupalKernel.php(549): Drupal\Component\DependencyInjection\Container->get('cache.container')
#9 /home/wghm7504/falzon9/core/lib/Drupal/Core/DrupalKernel.php(894): Drupal\Core\DrupalKernel->getCachedContainerDefinition()
#10 /home/wghm7504/falzon9/core/lib/Drupal/Core/DrupalKernel.php(472): Drupal\Core\DrupalKernel->initializeContainer()
#11 /home/wghm7504/falzon9/core/lib/Drupal/Core/DrupalKernel.php(707): Drupal\Core\DrupalKernel->boot()
#12 /home/wghm7504/falzon9/index.php(19): Drupal\Core\DrupalKernel->handle(Object(Symfony\Component\HttpFoundation\Request))
#13 {main}
[25-Feb-2022 09:40:31 Europe/Paris] Drupal\Core\Database\DatabaseAccessDeniedException: SQLSTATE[HY000] [1045] Access denied for user 'wghm7504_root'@'localhost' (using password: YES) in /home/wghm7504/falzon9/core/lib/Drupal/Core/Database/Driver/mysql/Connection.php on line 197 #0 /home/wghm7504/falzon9/core/lib/Drupal/Core/Database/Database.php(387): Drupal\Core\Database\Driver\mysql\Connection::open(Array)
#1 /home/wghm7504/falzon9/core/lib/Drupal/Core/Database/Database.php(169): Drupal\Core\Database\Database::openConnection('default', 'default')
#2 [internal function]: Drupal\Core\Database\Database::getConnection('default')
#3 /home/wghm7504/falzon9/core/lib/Drupal/Component/DependencyInjection/PhpArrayContainer.php(79): call_user_func_array('Drupal\\Core\\Dat...', Array)
#4 /home/wghm7504/falzon9/core/lib/Drupal/Component/DependencyInjection/Container.php(176): Drupal\Component\DependencyInjection\PhpArrayContainer->createService(Array, 'database')
#5 /home/wghm7504/falzon9/core/lib/Drupal/Component/DependencyInjection/PhpArrayContainer.php(212): Drupal\Component\DependencyInjection\Container->get('database', 1)
#6 /home/wghm7504/falzon9/core/lib/Drupal/Component/DependencyInjection/PhpArrayContainer.php(62): Drupal\Component\DependencyInjection\PhpArrayContainer->resolveServicesAndParameters(Array)
#7 /home/wghm7504/falzon9/core/lib/Drupal/Component/DependencyInjection/Container.php(176): Drupal\Component\DependencyInjection\PhpArrayContainer->createService(Array, 'cache.container')
#8 /home/wghm7504/falzon9/core/lib/Drupal/Core/DrupalKernel.php(549): Drupal\Component\DependencyInjection\Container->get('cache.container')
#9 /home/wghm7504/falzon9/core/lib/Drupal/Core/DrupalKernel.php(894): Drupal\Core\DrupalKernel->getCachedContainerDefinition()
#10 /home/wghm7504/falzon9/core/lib/Drupal/Core/DrupalKernel.php(472): Drupal\Core\DrupalKernel->initializeContainer()
#11 /home/wghm7504/falzon9/core/lib/Drupal/Core/DrupalKernel.php(707): Drupal\Core\DrupalKernel->boot()
#12 /home/wghm7504/falzon9/index.php(19): Drupal\Core\DrupalKernel->handle(Object(Symfony\Component\HttpFoundation\Request))
#13 {main}
[25-Feb-2022 09:50:54 Europe/Paris] Drupal\Core\Database\DatabaseAccessDeniedException: SQLSTATE[HY000] [1045] Access denied for user 'wghm7504_root'@'localhost' (using password: YES) in /home/wghm7504/falzon9/core/lib/Drupal/Core/Database/Driver/mysql/Connection.php on line 197 #0 /home/wghm7504/falzon9/core/lib/Drupal/Core/Database/Database.php(387): Drupal\Core\Database\Driver\mysql\Connection::open(Array)
#1 /home/wghm7504/falzon9/core/lib/Drupal/Core/Database/Database.php(169): Drupal\Core\Database\Database::openConnection('default', 'default')
#2 [internal function]: Drupal\Core\Database\Database::getConnection('default')
#3 /home/wghm7504/falzon9/core/lib/Drupal/Component/DependencyInjection/PhpArrayContainer.php(79): call_user_func_array('Drupal\\Core\\Dat...', Array)
#4 /home/wghm7504/falzon9/core/lib/Drupal/Component/DependencyInjection/Container.php(176): Drupal\Component\DependencyInjection\PhpArrayContainer->createService(Array, 'database')
#5 /home/wghm7504/falzon9/core/lib/Drupal/Component/DependencyInjection/PhpArrayContainer.php(212): Drupal\Component\DependencyInjection\Container->get('database', 1)
#6 /home/wghm7504/falzon9/core/lib/Drupal/Component/DependencyInjection/PhpArrayContainer.php(62): Drupal\Component\DependencyInjection\PhpArrayContainer->resolveServicesAndParameters(Array)
#7 /home/wghm7504/falzon9/core/lib/Drupal/Component/DependencyInjection/Container.php(176): Drupal\Component\DependencyInjection\PhpArrayContainer->createService(Array, 'cache.container')
#8 /home/wghm7504/falzon9/core/lib/Drupal/Core/DrupalKernel.php(549): Drupal\Component\DependencyInjection\Container->get('cache.container')
#9 /home/wghm7504/falzon9/core/lib/Drupal/Core/DrupalKernel.php(894): Drupal\Core\DrupalKernel->getCachedContainerDefinition()
#10 /home/wghm7504/falzon9/core/lib/Drupal/Core/DrupalKernel.php(472): Drupal\Core\DrupalKernel->initializeContainer()
#11 /home/wghm7504/falzon9/core/lib/Drupal/Core/DrupalKernel.php(707): Drupal\Core\DrupalKernel->boot()
#12 /home/wghm7504/falzon9/index.php(19): Drupal\Core\DrupalKernel->handle(Object(Symfony\Component\HttpFoundation\Request))
#13 {main}
[25-Feb-2022 09:50:54 Europe/Paris] Drupal\Core\Database\DatabaseAccessDeniedException: SQLSTATE[HY000] [1045] Access denied for user 'wghm7504_root'@'localhost' (using password: YES) in /home/wghm7504/falzon9/core/lib/Drupal/Core/Database/Driver/mysql/Connection.php on line 197 #0 /home/wghm7504/falzon9/core/lib/Drupal/Core/Database/Database.php(387): Drupal\Core\Database\Driver\mysql\Connection::open(Array)
#1 /home/wghm7504/falzon9/core/lib/Drupal/Core/Database/Database.php(169): Drupal\Core\Database\Database::openConnection('default', 'default')
#2 [internal function]: Drupal\Core\Database\Database::getConnection('default')
#3 /home/wghm7504/falzon9/core/lib/Drupal/Component/DependencyInjection/PhpArrayContainer.php(79): call_user_func_array('Drupal\\Core\\Dat...', Array)
#4 /home/wghm7504/falzon9/core/lib/Drupal/Component/DependencyInjection/Container.php(176): Drupal\Component\DependencyInjection\PhpArrayContainer->createService(Array, 'database')
#5 /home/wghm7504/falzon9/core/lib/Drupal/Component/DependencyInjection/PhpArrayContainer.php(212): Drupal\Component\DependencyInjection\Container->get('database', 1)
#6 /home/wghm7504/falzon9/core/lib/Drupal/Component/DependencyInjection/PhpArrayContainer.php(62): Drupal\Component\DependencyInjection\PhpArrayContainer->resolveServicesAndParameters(Array)
#7 /home/wghm7504/falzon9/core/lib/Drupal/Component/DependencyInjection/Container.php(176): Drupal\Component\DependencyInjection\PhpArrayContainer->createService(Array, 'cache.container')
#8 /home/wghm7504/falzon9/core/lib/Drupal/Core/DrupalKernel.php(549): Drupal\Component\DependencyInjection\Container->get('cache.container')
#9 /home/wghm7504/falzon9/core/lib/Drupal/Core/DrupalKernel.php(894): Drupal\Core\DrupalKernel->getCachedContainerDefinition()
#10 /home/wghm7504/falzon9/core/lib/Drupal/Core/DrupalKernel.php(472): Drupal\Core\DrupalKernel->initializeContainer()
#11 /home/wghm7504/falzon9/core/lib/Drupal/Core/DrupalKernel.php(707): Drupal\Core\DrupalKernel->boot()
#12 /home/wghm7504/falzon9/index.php(19): Drupal\Core\DrupalKernel->handle(Object(Symfony\Component\HttpFoundation\Request))
#13 {main}
[26-Feb-2022 09:57:00 Europe/Paris] TypeError: Argument 2 passed to shared_form_user_login_form_alter() must be an instance of FormStateInterface, instance of Drupal\Core\Form\FormState given, called in /home/wghm7504/falzon9/core/lib/Drupal/Core/Extension/ModuleHandler.php on line 539 in /home/wghm7504/falzon9/modules/custom/shared/shared.module on line 90 #0 /home/wghm7504/falzon9/core/lib/Drupal/Core/Extension/ModuleHandler.php(539): shared_form_user_login_form_alter(Array, Object(Drupal\Core\Form\FormState), 'user_login_form')
#1 /home/wghm7504/falzon9/core/lib/Drupal/Core/Form/FormBuilder.php(835): Drupal\Core\Extension\ModuleHandler->alter('form', Array, Object(Drupal\Core\Form\FormState), 'user_login_form')
#2 /home/wghm7504/falzon9/core/lib/Drupal/Core/Form/FormBuilder.php(279): Drupal\Core\Form\FormBuilder->prepareForm('user_login_form', Array, Object(Drupal\Core\Form\FormState))
#3 /home/wghm7504/falzon9/core/lib/Drupal/Core/Controller/FormController.php(73): Drupal\Core\Form\FormBuilder->buildForm(Object(Drupal\user\Form\UserLoginForm), Object(Drupal\Core\Form\FormState))
#4 [internal function]: Drupal\Core\Controller\FormController->getContentResult(Object(Symfony\Component\HttpFoundation\Request), Object(Drupal\Core\Routing\RouteMatch))
#5 /home/wghm7504/falzon9/core/lib/Drupal/Core/EventSubscriber/EarlyRenderingControllerWrapperSubscriber.php(123): call_user_func_array(Array, Array)
#6 /home/wghm7504/falzon9/core/lib/Drupal/Core/Render/Renderer.php(564): Drupal\Core\EventSubscriber\EarlyRenderingControllerWrapperSubscriber->Drupal\Core\EventSubscriber\{closure}()
#7 /home/wghm7504/falzon9/core/lib/Drupal/Core/EventSubscriber/EarlyRenderingControllerWrapperSubscriber.php(124): Drupal\Core\Render\Renderer->executeInRenderContext(Object(Drupal\Core\Render\RenderContext), Object(Closure))
#8 /home/wghm7504/falzon9/core/lib/Drupal/Core/EventSubscriber/EarlyRenderingControllerWrapperSubscriber.php(97): Drupal\Core\EventSubscriber\EarlyRenderingControllerWrapperSubscriber->wrapControllerExecutionInRenderContext(Array, Array)
#9 /home/wghm7504/falzon9/vendor/symfony/http-kernel/HttpKernel.php(158): Drupal\Core\EventSubscriber\EarlyRenderingControllerWrapperSubscriber->Drupal\Core\EventSubscriber\{closure}()
#10 /home/wghm7504/falzon9/vendor/symfony/http-kernel/HttpKernel.php(80): Symfony\Component\HttpKernel\HttpKernel->handleRaw(Object(Symfony\Component\HttpFoundation\Request), 1)
#11 /home/wghm7504/falzon9/core/lib/Drupal/Core/StackMiddleware/Session.php(58): Symfony\Component\HttpKernel\HttpKernel->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true)
#12 /home/wghm7504/falzon9/core/lib/Drupal/Core/StackMiddleware/KernelPreHandle.php(48): Drupal\Core\StackMiddleware\Session->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true)
#13 /home/wghm7504/falzon9/core/modules/page_cache/src/StackMiddleware/PageCache.php(191): Drupal\Core\StackMiddleware\KernelPreHandle->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true)
#14 /home/wghm7504/falzon9/core/modules/page_cache/src/StackMiddleware/PageCache.php(128): Drupal\page_cache\StackMiddleware\PageCache->fetch(Object(Symfony\Component\HttpFoundation\Request), 1, true)
#15 /home/wghm7504/falzon9/core/modules/page_cache/src/StackMiddleware/PageCache.php(82): Drupal\page_cache\StackMiddleware\PageCache->lookup(Object(Symfony\Component\HttpFoundation\Request), 1, true)
#16 /home/wghm7504/falzon9/core/lib/Drupal/Core/StackMiddleware/ReverseProxyMiddleware.php(48): Drupal\page_cache\StackMiddleware\PageCache->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true)
#17 /home/wghm7504/falzon9/core/lib/Drupal/Core/StackMiddleware/NegotiationMiddleware.php(51): Drupal\Core\StackMiddleware\ReverseProxyMiddleware->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true)
#18 /home/wghm7504/falzon9/vendor/stack/builder/src/Stack/StackedHttpKernel.php(23): Drupal\Core\StackMiddleware\NegotiationMiddleware->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true)
#19 /home/wghm7504/falzon9/core/lib/Drupal/Core/DrupalKernel.php(708): Stack\StackedHttpKernel->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true)
#20 /home/wghm7504/falzon9/index.php(19): Drupal\Core\DrupalKernel->handle(Object(Symfony\Component\HttpFoundation\Request))
#21 {main}
[26-Feb-2022 09:58:21 Europe/Paris] Error: Class 'Url' not found in /home/wghm7504/falzon9/modules/custom/shared/shared.module on line 99 #0 [internal function]: shared_user_login_form_submit(Array, Object(Drupal\Core\Form\FormState))
#1 /home/wghm7504/falzon9/core/lib/Drupal/Core/Form/FormSubmitter.php(114): call_user_func_array('shared_user_log...', Array)
#2 /home/wghm7504/falzon9/core/lib/Drupal/Core/Form/FormSubmitter.php(52): Drupal\Core\Form\FormSubmitter->executeSubmitHandlers(Array, Object(Drupal\Core\Form\FormState))
#3 /home/wghm7504/falzon9/core/lib/Drupal/Core/Form/FormBuilder.php(592): Drupal\Core\Form\FormSubmitter->doSubmitForm(Array, Object(Drupal\Core\Form\FormState))
#4 /home/wghm7504/falzon9/core/lib/Drupal/Core/Form/FormBuilder.php(320): Drupal\Core\Form\FormBuilder->processForm('user_login_form', Array, Object(Drupal\Core\Form\FormState))
#5 /home/wghm7504/falzon9/core/lib/Drupal/Core/Controller/FormController.php(73): Drupal\Core\Form\FormBuilder->buildForm(Object(Drupal\user\Form\UserLoginForm), Object(Drupal\Core\Form\FormState))
#6 [internal function]: Drupal\Core\Controller\FormController->getContentResult(Object(Symfony\Component\HttpFoundation\Request), Object(Drupal\Core\Routing\RouteMatch))
#7 /home/wghm7504/falzon9/core/lib/Drupal/Core/EventSubscriber/EarlyRenderingControllerWrapperSubscriber.php(123): call_user_func_array(Array, Array)
#8 /home/wghm7504/falzon9/core/lib/Drupal/Core/Render/Renderer.php(564): Drupal\Core\EventSubscriber\EarlyRenderingControllerWrapperSubscriber->Drupal\Core\EventSubscriber\{closure}()
#9 /home/wghm7504/falzon9/core/lib/Drupal/Core/EventSubscriber/EarlyRenderingControllerWrapperSubscriber.php(124): Drupal\Core\Render\Renderer->executeInRenderContext(Object(Drupal\Core\Render\RenderContext), Object(Closure))
#10 /home/wghm7504/falzon9/core/lib/Drupal/Core/EventSubscriber/EarlyRenderingControllerWrapperSubscriber.php(97): Drupal\Core\EventSubscriber\EarlyRenderingControllerWrapperSubscriber->wrapControllerExecutionInRenderContext(Array, Array)
#11 /home/wghm7504/falzon9/vendor/symfony/http-kernel/HttpKernel.php(158): Drupal\Core\EventSubscriber\EarlyRenderingControllerWrapperSubscriber->Drupal\Core\EventSubscriber\{closure}()
#12 /home/wghm7504/falzon9/vendor/symfony/http-kernel/HttpKernel.php(80): Symfony\Component\HttpKernel\HttpKernel->handleRaw(Object(Symfony\Component\HttpFoundation\Request), 1)
#13 /home/wghm7504/falzon9/core/lib/Drupal/Core/StackMiddleware/Session.php(58): Symfony\Component\HttpKernel\HttpKernel->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true)
#14 /home/wghm7504/falzon9/core/lib/Drupal/Core/StackMiddleware/KernelPreHandle.php(48): Drupal\Core\StackMiddleware\Session->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true)
#15 /home/wghm7504/falzon9/core/modules/page_cache/src/StackMiddleware/PageCache.php(106): Drupal\Core\StackMiddleware\KernelPreHandle->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true)
#16 /home/wghm7504/falzon9/core/modules/page_cache/src/StackMiddleware/PageCache.php(85): Drupal\page_cache\StackMiddleware\PageCache->pass(Object(Symfony\Component\HttpFoundation\Request), 1, true)
#17 /home/wghm7504/falzon9/core/lib/Drupal/Core/StackMiddleware/ReverseProxyMiddleware.php(48): Drupal\page_cache\StackMiddleware\PageCache->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true)
#18 /home/wghm7504/falzon9/core/lib/Drupal/Core/StackMiddleware/NegotiationMiddleware.php(51): Drupal\Core\StackMiddleware\ReverseProxyMiddleware->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true)
#19 /home/wghm7504/falzon9/vendor/stack/builder/src/Stack/StackedHttpKernel.php(23): Drupal\Core\StackMiddleware\NegotiationMiddleware->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true)
#20 /home/wghm7504/falzon9/core/lib/Drupal/Core/DrupalKernel.php(708): Stack\StackedHttpKernel->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true)
#21 /home/wghm7504/falzon9/index.php(19): Drupal\Core\DrupalKernel->handle(Object(Symfony\Component\HttpFoundation\Request))
#22 {main}
[26-Feb-2022 09:58:44 Europe/Paris] Error: Class 'Url' not found in /home/wghm7504/falzon9/modules/custom/shared/shared.module on line 99 #0 [internal function]: shared_user_login_form_submit(Array, Object(Drupal\Core\Form\FormState))
#1 /home/wghm7504/falzon9/core/lib/Drupal/Core/Form/FormSubmitter.php(114): call_user_func_array('shared_user_log...', Array)
#2 /home/wghm7504/falzon9/core/lib/Drupal/Core/Form/FormSubmitter.php(52): Drupal\Core\Form\FormSubmitter->executeSubmitHandlers(Array, Object(Drupal\Core\Form\FormState))
#3 /home/wghm7504/falzon9/core/lib/Drupal/Core/Form/FormBuilder.php(592): Drupal\Core\Form\FormSubmitter->doSubmitForm(Array, Object(Drupal\Core\Form\FormState))
#4 /home/wghm7504/falzon9/core/lib/Drupal/Core/Form/FormBuilder.php(320): Drupal\Core\Form\FormBuilder->processForm('user_login_form', Array, Object(Drupal\Core\Form\FormState))
#5 /home/wghm7504/falzon9/core/lib/Drupal/Core/Controller/FormController.php(73): Drupal\Core\Form\FormBuilder->buildForm(Object(Drupal\user\Form\UserLoginForm), Object(Drupal\Core\Form\FormState))
#6 [internal function]: Drupal\Core\Controller\FormController->getContentResult(Object(Symfony\Component\HttpFoundation\Request), Object(Drupal\Core\Routing\RouteMatch))
#7 /home/wghm7504/falzon9/core/lib/Drupal/Core/EventSubscriber/EarlyRenderingControllerWrapperSubscriber.php(123): call_user_func_array(Array, Array)
#8 /home/wghm7504/falzon9/core/lib/Drupal/Core/Render/Renderer.php(564): Drupal\Core\EventSubscriber\EarlyRenderingControllerWrapperSubscriber->Drupal\Core\EventSubscriber\{closure}()
#9 /home/wghm7504/falzon9/core/lib/Drupal/Core/EventSubscriber/EarlyRenderingControllerWrapperSubscriber.php(124): Drupal\Core\Render\Renderer->executeInRenderContext(Object(Drupal\Core\Render\RenderContext), Object(Closure))
#10 /home/wghm7504/falzon9/core/lib/Drupal/Core/EventSubscriber/EarlyRenderingControllerWrapperSubscriber.php(97): Drupal\Core\EventSubscriber\EarlyRenderingControllerWrapperSubscriber->wrapControllerExecutionInRenderContext(Array, Array)
#11 /home/wghm7504/falzon9/vendor/symfony/http-kernel/HttpKernel.php(158): Drupal\Core\EventSubscriber\EarlyRenderingControllerWrapperSubscriber->Drupal\Core\EventSubscriber\{closure}()
#12 /home/wghm7504/falzon9/vendor/symfony/http-kernel/HttpKernel.php(80): Symfony\Component\HttpKernel\HttpKernel->handleRaw(Object(Symfony\Component\HttpFoundation\Request), 1)
#13 /home/wghm7504/falzon9/core/lib/Drupal/Core/StackMiddleware/Session.php(58): Symfony\Component\HttpKernel\HttpKernel->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true)
#14 /home/wghm7504/falzon9/core/lib/Drupal/Core/StackMiddleware/KernelPreHandle.php(48): Drupal\Core\StackMiddleware\Session->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true)
#15 /home/wghm7504/falzon9/core/modules/page_cache/src/StackMiddleware/PageCache.php(106): Drupal\Core\StackMiddleware\KernelPreHandle->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true)
#16 /home/wghm7504/falzon9/core/modules/page_cache/src/StackMiddleware/PageCache.php(85): Drupal\page_cache\StackMiddleware\PageCache->pass(Object(Symfony\Component\HttpFoundation\Request), 1, true)
#17 /home/wghm7504/falzon9/core/lib/Drupal/Core/StackMiddleware/ReverseProxyMiddleware.php(48): Drupal\page_cache\StackMiddleware\PageCache->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true)
#18 /home/wghm7504/falzon9/core/lib/Drupal/Core/StackMiddleware/NegotiationMiddleware.php(51): Drupal\Core\StackMiddleware\ReverseProxyMiddleware->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true)
#19 /home/wghm7504/falzon9/vendor/stack/builder/src/Stack/StackedHttpKernel.php(23): Drupal\Core\StackMiddleware\NegotiationMiddleware->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true)
#20 /home/wghm7504/falzon9/core/lib/Drupal/Core/DrupalKernel.php(708): Stack\StackedHttpKernel->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true)
#21 /home/wghm7504/falzon9/index.php(19): Drupal\Core\DrupalKernel->handle(Object(Symfony\Component\HttpFoundation\Request))
#22 {main}
[26-Feb-2022 09:58:57 Europe/Paris] Error: Class 'Url' not found in /home/wghm7504/falzon9/modules/custom/shared/shared.module on line 99 #0 [internal function]: shared_user_login_form_submit(Array, Object(Drupal\Core\Form\FormState))
#1 /home/wghm7504/falzon9/core/lib/Drupal/Core/Form/FormSubmitter.php(114): call_user_func_array('shared_user_log...', Array)
#2 /home/wghm7504/falzon9/core/lib/Drupal/Core/Form/FormSubmitter.php(52): Drupal\Core\Form\FormSubmitter->executeSubmitHandlers(Array, Object(Drupal\Core\Form\FormState))
#3 /home/wghm7504/falzon9/core/lib/Drupal/Core/Form/FormBuilder.php(592): Drupal\Core\Form\FormSubmitter->doSubmitForm(Array, Object(Drupal\Core\Form\FormState))
#4 /home/wghm7504/falzon9/core/lib/Drupal/Core/Form/FormBuilder.php(320): Drupal\Core\Form\FormBuilder->processForm('user_login_form', Array, Object(Drupal\Core\Form\FormState))
#5 /home/wghm7504/falzon9/core/lib/Drupal/Core/Controller/FormController.php(73): Drupal\Core\Form\FormBuilder->buildForm(Object(Drupal\user\Form\UserLoginForm), Object(Drupal\Core\Form\FormState))
#6 [internal function]: Drupal\Core\Controller\FormController->getContentResult(Object(Symfony\Component\HttpFoundation\Request), Object(Drupal\Core\Routing\RouteMatch))
#7 /home/wghm7504/falzon9/core/lib/Drupal/Core/EventSubscriber/EarlyRenderingControllerWrapperSubscriber.php(123): call_user_func_array(Array, Array)
#8 /home/wghm7504/falzon9/core/lib/Drupal/Core/Render/Renderer.php(564): Drupal\Core\EventSubscriber\EarlyRenderingControllerWrapperSubscriber->Drupal\Core\EventSubscriber\{closure}()
#9 /home/wghm7504/falzon9/core/lib/Drupal/Core/EventSubscriber/EarlyRenderingControllerWrapperSubscriber.php(124): Drupal\Core\Render\Renderer->executeInRenderContext(Object(Drupal\Core\Render\RenderContext), Object(Closure))
#10 /home/wghm7504/falzon9/core/lib/Drupal/Core/EventSubscriber/EarlyRenderingControllerWrapperSubscriber.php(97): Drupal\Core\EventSubscriber\EarlyRenderingControllerWrapperSubscriber->wrapControllerExecutionInRenderContext(Array, Array)
#11 /home/wghm7504/falzon9/vendor/symfony/http-kernel/HttpKernel.php(158): Drupal\Core\EventSubscriber\EarlyRenderingControllerWrapperSubscriber->Drupal\Core\EventSubscriber\{closure}()
#12 /home/wghm7504/falzon9/vendor/symfony/http-kernel/HttpKernel.php(80): Symfony\Component\HttpKernel\HttpKernel->handleRaw(Object(Symfony\Component\HttpFoundation\Request), 1)
#13 /home/wghm7504/falzon9/core/lib/Drupal/Core/StackMiddleware/Session.php(58): Symfony\Component\HttpKernel\HttpKernel->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true)
#14 /home/wghm7504/falzon9/core/lib/Drupal/Core/StackMiddleware/KernelPreHandle.php(48): Drupal\Core\StackMiddleware\Session->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true)
#15 /home/wghm7504/falzon9/core/modules/page_cache/src/StackMiddleware/PageCache.php(106): Drupal\Core\StackMiddleware\KernelPreHandle->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true)
#16 /home/wghm7504/falzon9/core/modules/page_cache/src/StackMiddleware/PageCache.php(85): Drupal\page_cache\StackMiddleware\PageCache->pass(Object(Symfony\Component\HttpFoundation\Request), 1, true)
#17 /home/wghm7504/falzon9/core/lib/Drupal/Core/StackMiddleware/ReverseProxyMiddleware.php(48): Drupal\page_cache\StackMiddleware\PageCache->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true)
#18 /home/wghm7504/falzon9/core/lib/Drupal/Core/StackMiddleware/NegotiationMiddleware.php(51): Drupal\Core\StackMiddleware\ReverseProxyMiddleware->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true)
#19 /home/wghm7504/falzon9/vendor/stack/builder/src/Stack/StackedHttpKernel.php(23): Drupal\Core\StackMiddleware\NegotiationMiddleware->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true)
#20 /home/wghm7504/falzon9/core/lib/Drupal/Core/DrupalKernel.php(708): Stack\StackedHttpKernel->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true)
#21 /home/wghm7504/falzon9/index.php(19): Drupal\Core\DrupalKernel->handle(Object(Symfony\Component\HttpFoundation\Request))
#22 {main}
